src/Components/Web/src/Virtualization/VirtualizeFixed.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Components.Rendering;
+
+namespace Microsoft.AspNetCore.Components.Virtualization
+{
+    /// <summary>
+    /// Provides functionality for rendering a virtualized fixed list of items.
```

Comment:
How would it look if we wanted to merge `VirtualizeDeferred` and `VirtualizeFixed` into one thing? I know you will have split them apart for a reason, but want to clarify what since this would be public API.

Is it possible that `VirtualizeFixed` could reduce to a special case within `VirtualizeDeferred`, in which the data provider just happens to return a `Task<ItemsProviderResult>` that is always already completed synchronously?

I guess the merged component would have to have both `Items` and `ItemsProvider` parameters and we'd need a runtime check to tell the developer not to supply both, but that seems like a reasonable tradeoff for the benefit of not having to teach people about two different base classes.

Reply:
I've implemented a data fetching flow similar to what you suggested, which made the task of combining the two easier. Thanks for the suggestion, this has now been done!